data/net/minecraft/tags/TagCollection.mapping

```diff
@@ -7,10 +12,21 @@ CLASS net/minecraft/tags/TagCollection
 		ARG 1 resourceLocation
 	METHOD getTagOrEmpty (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/tags/Tag;
 		ARG 1 id
+	METHOD hasTag (Lnet/minecraft/resources/ResourceLocation;)Z
+		ARG 1 resourceLocation
 	METHOD of (Ljava/util/Map;)Lnet/minecraft/tags/TagCollection;
 		ARG 0 idTagMap
+	METHOD serializeToNetwork (Lnet/minecraft/core/Registry;)Lnet/minecraft/tags/TagCollection$NetworkPayload;
+		ARG 1 registry
 	CLASS 1
 		METHOD getId (Lnet/minecraft/tags/Tag;)Lnet/minecraft/resources/ResourceLocation;
 			ARG 1 tag
 		METHOD getTagOrEmpty (Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/tags/Tag;
 			ARG 1 id
+	CLASS NetworkPayload
+		METHOD <init> (Ljava/util/Map;)V
+			ARG 1 tags
+		METHOD read (Lnet/minecraft/network/FriendlyByteBuf;)Lnet/minecraft/tags/TagCollection$NetworkPayload;
+			ARG 0 byteBuf
```

Comment:
Needs to be discussed on whether to have all buffers as `byteBuf` or `buffer`.

Reply:
Personally I like `buffer`, but when I was doing it I felt that using `byteBuf` would probably be safer and clearer as to what type of buffer